### PR TITLE
adds route for categories

### PR DESCRIPTION
**Pull request by Vladomira**
возможно надо что-то с ошибками доработать.

### CODE REVIEW

File: controllers/categoriesCtrl.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const operations = require('../models/category')
```
Comment (Prykhodko-Pasha) [inline controllers/categoriesCtrl.js]: Why it's called "operations"? 
Maybe you don't need a model of category at all? You can read and parse json-file wright here in controller

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "cross-env": "7.0.3",
     "dotenv": "10.0.0",
     "express": "4.17.1",
+    "http-errors": "^1.7.0",
```
Comment (Prykhodko-Pasha) [inline package.json]: Please delete a caret

File: controllers/categoriesCtrl.js (hunk 1 above)
Comment (Vladomira) [reply controllers/categoriesCtrl.js]: operations - because it is manipulations with data from files
fixed

File: package.json (hunk 2 above)
Comment (Vladomira) [reply package.json]: deleted

**Review by Prykhodko-Pasha — APPROVED** [review]

**Review by SovaV — APPROVED** [review]

**Pull request merged**